### PR TITLE
Update 'meetings' command

### PR DESCRIPTION
**Pull request by jimseah-0116**
'meetings' command will now only display this week's meetings.

### CODE REVIEW

**Review by youdonnnn — APPROVED** [review]
LGTM! Good work on changing the meetings tab to display only this week's meetings.

File: src/main/java/seedu/address/model/person/Meeting.java [hunk 1]
```diff
@@ -53,7 +55,11 @@ private static LocalDateTime stringToDateTime(String dateTime) {
         if (dateTime.length() != 16) {
             throw new IllegalArgumentException("The date and time must be in the format yyyy-MM-dd HH:mm");
         }
-        return LocalDateTime.parse(dateTime, DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm"));
+        try {
```
Comment (Greg-Tan) [inline src/main/java/seedu/address/model/person/Meeting.java:58]: nice job catching exceptions

File: src/main/java/seedu/address/ui/MeetingsWindow.java [hunk 2]
```diff
@@ -61,21 +63,23 @@ public MeetingsWindow(Logic logic) {
      *     </ul>
      */
     public void displayMeetings() {
-        logger.fine("Showing meetings window for the application.");
+        logger.fine("Showing this week's meetings window for the application.");
```
Comment (Greg-Tan) [inline src/main/java/seedu/address/ui/MeetingsWindow.java:66]: nice job specifying the time period

File: src/main/java/seedu/address/ui/MeetingsWindow.java [hunk 3]
```diff
@@ -61,21 +63,23 @@ public MeetingsWindow(Logic logic) {
...
         getRoot().show();
         getRoot().centerOnScreen();
 
-        List<Person> people = logic.getFilteredPersonList();
-        people = people.stream()
-                .filter(p -> p.getMeeting() != null)
+        List<Meeting> meetings = logic.getMeetingList();
```
Comment (Greg-Tan) [inline src/main/java/seedu/address/ui/MeetingsWindow.java:70]: this seems to work better!

**Review by Greg-Tan — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/person/Meeting.java (hunk 1 above)
Comment (irwynliong) [inline src/main/java/seedu/address/model/person/Meeting.java:58]: good use of try catch here!

File: src/main/java/seedu/address/model/person/Policy.java [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 /**
  * Represents a policy associated with a person.
- * Guarantees: immutable; policyName is valid as declared in {@link #isValidField(String)}
+ * Guarantees: immutable; policyName is valid as declared in {@link #isValidPolicy(String)}
```
Comment (irwynliong) [inline src/main/java/seedu/address/model/person/Policy.java:12]: good catch of error on our part! 👍

**Review by irwynliong — APPROVED** [review]
LGTM! Thanks for the work

**Pull request merged**